app/templates/views/all_galleries.html

```diff
@@ -1,54 +1,52 @@
 {% extends "snips/base.html" %}
 {% block body %}
+
+<link rel="stylesheet" href="/static/css/all_galleries.css" type="text/css" />
+
+
 <div class="container">
 <h2>All Galleries View</h2>
 </div>
+
 <div class="container">
-<a href='/application/create'></href><button float="left" type="button" class="btn btn-primary" style="float: right;">Create</button><a/>
-<a href='/gallery/submission'><button float="left" type="button" class="btn btn-primary" style="float: left;">Submissions</button></a>
+<a href='/application/create/1' ></href> <button type="button" id ="create" style="float:right"  class="btn btn-primary" >Create</button><a/>
+<a href='/gallery/submission' ><button type="button" id ="submit"class="btn btn-primary">Submissions</button></a>
 </div>
+
 <h2 align="center">Galleries</h2> 
 
-<div class="container">
-  <table class="table table-bordered">
-    <thead>
-      <tr>
-        <th>Gallery</th>
-        <th>Number of Submissions</th>
-        <th>Deadline</th>
-         <th> Edit</th>
-      </tr>
+<div class ="container">
+
+<table id="gallery" class= "table table-bordered">
+
+<thead>
+       <tr>
+            <th>Galleries</th> 
+            <th>Submissions</th>
+            <th>Status</th>
+            <th>Edit</th>
+        </tr> 
     </thead>
```

Comment:
Please format html so that it is readable. 
Here the </thead> is intented too much
It is important since formatting makes it easier to find mistakes, for example the div on line 18 is never closed, but it is hard to tell since table on line 50 is in line with the div

Reply:
Hey guys @memo330179  and @Jessonsotoventura  Of my latest commits, ignore this one. I wasnt supposed to commit this file. 
![image](https://user-images.githubusercontent.com/22758750/35609441-539dce48-062b-11e8-82b4-0cfc47f86ddb.png)